src/models/fields/IllnessStartDate.schema.ts

```diff
@@ -0,0 +1,50 @@
+import * as Joi from '@hapi/joi';
+
+const illnessStartDayErrorMessage: string = 'You must enter a day';
+const illnessStartMonthErrorMessage: string = 'You must enter a month';
+const illnessStartYearErrorMessage: string = 'You must enter a year';
+const invalidDate: string = 'Enter a real date';
+const invalidDateFuture: string = 'Start date must be today or in the past';
+
+const dayMonthRegex: RegExp = /^[0-9]{1,2}$/i;
+const yearRegex: RegExp = /^[0-9]{2,4}$/i;
```

Comment:
do we really want to allow 2 characters for years?

Reply:
(if we are meant to allow for years with only 2 characters, could we have a test for that please)

Reply:
i'm not 100% sure about this.. the example does suggest 4 characters - i'll follow the example as that's all we have